src/tools/kata-ctl/src/arch/x86_64/mod.rs

```diff
@@ -19,12 +20,41 @@ mod arch_specific {
     pub const ARCH_CPU_MODEL_FIELD: &str = check::GENERIC_CPU_MODEL_FIELD;
 
     // List of check functions
-    static CHECK_LIST: &[CheckItem] = &[CheckItem {
-        name: CheckType::CheckCpu,
-        descr: "This parameter performs the cpu check",
-        fp: check_cpu,
-        perm: PermissionType::NonPrivileged,
-    }];
+    static CHECK_LIST: &[CheckItem] = &[
+        CheckItem {
+            name: CheckType::CheckCpu,
+            descr: "This parameter performs the cpu check",
+            fp: check_cpu,
+            perm: PermissionType::NonPrivileged,
+        },
+        CheckItem {
+            name: CheckType::CheckKernelModules,
+            descr: "This parameter performs the kvm check",
+            fp: check_kernel_modules,
+            perm: PermissionType::NonPrivileged,
+        },
+    ];
```

Comment:
@dborquez, FYI, when I added a 3rd CheckType I got a compiler warning about having the same prefix, so my PR will change these to Cpu and KernelModules from CheckCpu and CheckKernelModules, but you could change it now if you want

Reply:
@cmaf do you want to change `CheckKernelModules `to `KernelModules `?